### DISCUSSION TITLE
/the-basics/grpc.html

### DISCUSSION THREAD

**Discussion by giscus** [goravel/goravel#81] [Announcements]
# /the-basics/grpc.html

[[toc]] Introduction Grpc module can be operated by facades.Grpc. Controllers Controllers can be defined in the /app/grpc/controllers directory. Define routing All routing files...

https://www.goravel.dev/the-basics/grpc.html

**Comment by devig** [comment]
Would be nice to have an example

**Reply by hwbrzzl** [reply]
Sure, I'll add one in [goravel/example](https://github.com/goravel/example)

**Reply by hwbrzzl** [reply]
Hi @devig I added an example, PTAL: https://github.com/goravel/example#grpc
